tests/format/typescript/argument-expansion/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -85,3 +85,75 @@ const bar8 = [1, 2, 3].reduce(
 
 ================================================================================
 `;
+
+exports[`arrow-with-return-type.ts format 1`] = `
+====================================options=====================================
+parsers: ["typescript"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+longfunctionWithCall1("bla", foo, (thing: string): complex<type<something>> => {
+  code();
+});
+
+longfunctionWithCall12("bla", foo, (thing: string): complex<type<something>> => {
+  code();
+});
+
+longfunctionWithCallBack("blabla", foobarbazblablablablabla, (thing: string): complex<type<something>> => {
+  code();
+});
+
+longfunctionWithCallBack("blabla", foobarbazblablabla, (thing: string): complex<type<something>> => {
+  code();
+});
+
+longfunctionWithCall1("bla", foo, (thing: string): complex<type<\`
+\`>> => {
+  code();
+});
+
+=====================================output=====================================
+longfunctionWithCall1("bla", foo, (thing: string): complex<type<something>> => {
+  code();
+});
+
+longfunctionWithCall12(
+  "bla",
+  foo,
+  (thing: string): complex<type<something>> => {
+    code();
+  }
+);
+
+longfunctionWithCallBack(
+  "blabla",
+  foobarbazblablablablabla,
+  (thing: string): complex<type<something>> => {
+    code();
+  }
+);
+
+longfunctionWithCallBack(
+  "blabla",
+  foobarbazblablabla,
+  (thing: string): complex<type<something>> => {
+    code();
+  }
+);
+
+longfunctionWithCall1(
+  "bla",
+  foo,
+  (
+    thing: string
+  ): complex<
```

Comment:
It may be a trivial matter, but this line breaks is expected?

Reply:
It's not, but it's a separate issue

Reply:
ok